requirements/portable-exe-extras-win-lock.txt

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Do we use this lock file anywhere? I would assume we should be able to remove this and rely solely on the `portable-exe-win-lock.txt` file for download deps.